guide/popular-topics/faq.md

```diff
@@ -2,20 +2,20 @@
 
 ## Legend
 
-* `client` is a placeholder for the <DocsLink path="class/Client" /> object, such as `const client = new Client({ intents: [Intents.FLAGS.GUILDS] });`.
+* `client` is a placeholder for the <DocsLink path="class/Client" /> object, such as `const client = new Client({ intents: [GatewayIntentBits.Guilds] });`.
 * `interaction` is a placeholder for the <DocsLink path="class/Interaction" /> object, such as `client.on('interactionCreate', interaction => { ... });`.
 * `guild` is a placeholder for the <DocsLink path="class/Guild" /> object, such as `interaction.guild` or `client.guilds.cache.get('id')`.
-* `voiceChannel` is a placeholder for the <DocsLink path="class/VoiceChannel" /> object, such as `message.member.voice.channel`
+* `voiceChannel` is a placeholder for the <DocsLink path="class/VoiceChannel" /> object, such as `interaction.member.voice.channel`
 
 For a more detailed explanation of the notations commonly used in this guide, the docs, and the support server, see [here](/additional-info/notation.md).
 
 ## Administrative
 
-### How do I ban a user?
+### How do I ban a Guild Member?
 
 ```js
-const user = interaction.options.getUser('target');
-guild.members.ban(user);
+const member = interaction.options.getMember('target');
+member.ban();
```

Comment:
This should be reverted. The implication here is being able to ban those that are not in the server.